python/setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
       name='dpu_utils',
-      version='0.2.7',
+      version='0.2.8',

```

Comment:
I kind of feel API-breaking changes should generate a major version bump; we haven't been very consistent about this anyway, but maybe it's time to be?

Reply:
You mean go to version `1.0`, right? If so, it might be worth thinking if there are any other breaking API changes that we need to make here..